src/python/grpcio/grpc/_channel.py

```diff
@@ -969,4 +974,5 @@ def stream_stream(self,
             _common.encode(method), request_serializer, response_deserializer)
 
     def __del__(self):
+        _unsubscribe(self._connectivity_state, self._dummy_callback)
```

Comment:
How certain are you that this is necessary? What does this do that the `_moot` call on the next line doesn't also do? If we can eliminate this call, we can also eliminate the maintenance of the dummy callback as an instance field, right?